assets/settings/default.json

```diff
@@ -83,6 +83,9 @@
   "confirm_quit": false,
   // Whether to restore last closed project when fresh Zed instance is opened.
   "restore_on_startup": "last_workspace",
+  // Whether zed should automatically download binaries to run
+  // language servers if they are missing.
+  "download_missing_binaries": true,
```

Comment:
This should likely default to false, to ensure that there is a requirement for user explicit consent

Reply:
We will not be defaulting to `false`, as this provides a poor UX out of the box.

This setting is for users who have an issue with the default behavior to opt out of it.

Reply:
Understood, I'm only providing feedback. A default of false is not particularly useful at all though, it's like building an authentication system that by default does not need a user to login. Companies are restricting use of zed due to the lack of explicit consent. I personally absolutely love Zed and think it'd be a shame if it grew out of use due to security concerns